.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Integration tests
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache Node.js modules
+        uses: actions/cache@v4
+        with:
+          path: ./integration-tests/chopsticks/node_modules
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/integration-tests/chopsticks/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-modules-
+
+      - name: Set up Node.js environment
+        run: |
+          cd ./integration-tests/chopsticks
```

Comment:
Also, can we set a workdir that applies to all steps, or not? We're always operating within the `integration-tests/chopstics` dir, which is kinda a project on its own.

Reply:
Added

Reply:
I meant something like [this](https://github.com/orgs/community/discussions/25742#discussioncomment-4013289)

Reply:
Done